android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/ConnectionHealth.kt

```diff
@@ -21,12 +21,38 @@ class ConnectionHealth(service: VPNService) {
     private var mGateway: String = ""
     private var mAltEndpoint: String = ""
     private var mResetUsed = false
+    private var mPanicStateReached = false
 
     var mActive = false
     var mVPNNetwork: Network? = null
     var mWorker: ExecutorService = Executors.newSingleThreadExecutor()
 
+    @Deprecated("Only added temporary telemetry, please remove c:")
+    fun getStatusString(): String {
+        if (!mActive) {
+            // We have been feature flagged-off, or are disconnected

```

Comment:
Woul  it be beneficial to have a different state for for "feature flagged off"? Is that even possible?

Reply:
Added :)

Reply:
The comment needs to be updated then.